larray/core/session.py

```diff
@@ -24,6 +25,26 @@ class Session(object):
         Name of file to load or dictionary containing couples (name, array).
     kwargs : dict of str, array
         List of arrays to add written as 'name'=array, ...
+        
+    Examples
+    --------
+    >>> arr1, arr2, arr3 = ndtest((3, 3)), ndtest((6, 4)), ndtest((2, 8))
```

Comment:
general comment for all doctests: I would prefer using (even) smaller arrays such as (2, 3), (3) and (2, 2)

* repr takes less space and is faster for a human to "parse"
* doctests execute a tad faster